pipeline/export/nwb.py

```diff
@@ -6,11 +6,15 @@
 from dateutil.tz import tzlocal
 import pynwb
 from pynwb import NWBFile, NWBHDF5IO
+from pynwb.file import Subject
+from pynwb.behavior import SpatialSeries, Position
```

Comment:
you probably don't need these two, as there's already `pynwb` imported. You can use `pynwb.file.Subject` or `pynwb.behavior.SpatialSeries`

Reply:
I will adjust.